core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -744,9 +744,16 @@ public void sendTo(Address destination, ReplicableCommand rpcCommand, DeliverOrd
          }
          return;
       }
-      dispatcher.sendMessage(
+      CommandAwareRpcDispatcher rpcDispatcher = dispatcher;
+      if (rpcDispatcher == null) {
+         if (trace) {
+            log.trace("Don't send message. Transport is stopped.");
```

Comment:
why? if the ```dispatcher``` is ```null``` then the ```channel``` is already disconnected and the node left the view. There is no need to spam the log with this message.